libs/data-types/src/lib/validation/coscrad-data-type-validation-constraints/validate-coscrad-model-property.ts

```diff
@@ -44,6 +44,15 @@ const validateArray = (
     // Double check that if it's not optional it has length
     if (isOptional && isDeepStrictEqual(inputValue, [])) return [];
 
+    // TODO Add regression test in this lib
```

Comment:
We need a comprehensive test suite for this lib. We are only testing it at the integration level through our domain as far as I can tell.